db/schema.rb

```diff
@@ -54,25 +53,26 @@
     t.string "name"
     t.text "bio"
     t.string "tagline"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
     t.string "image_url"
+    t.jsonb "links", default: {}
```

Comment:
I expected that this would have been the only change to the schema, based on the migration that I did.